src/actions/index.test.js

```diff
@@ -0,0 +1,39 @@
+import * as actions from './index'
+
+describe('actions', () => {
+  it('should return a type of IS_LOADING, with isLoading', () => {
+    const isLoading = true
+    const expected = {
+      type: 'IS_LOADING',
+      isLoading
+    }
+
+    const result = actions.isLoading(isLoading)
+
+    expect(result).toEqual(expected)
+  })
+
+  it('should return a type of HAS_ERRORED, with an id', () => {
```

Comment:
Your implementation of this test looks great! The it block is a little misleading with stating 'with an id' when it is returning hasErrored.

Reply:
whoops, thanks for pointing that out! Fixed!